packages/core-utils/src/core-utils.story.tsx

```diff
@@ -138,54 +146,39 @@ export const RouteSortingLogic = (): JSX.Element => {
     stringLongName
   ].filter(x => x !== null);
 
-  function makeRouteComparator(): (a: number, b: number) => number {
-    return makeMultiCriteriaSort(
-      ...(sortArray as Array<(a: any, b: any) => number>)
-    );
-  }
-
   const sortedRoutes = Array.from(routes as Array<any>).sort(
-    makeRouteComparator()
+    makeRouteComparator(sortArray)
   );
 
   return (
     <Columns>
-      <div
-        style={{
-          display: "flex",
-          flexDirection: "column",
-
-          width: "100%"
-        }}
-      >
+      <StyledTable>
         <RouteRow>
```

Comment:
Convert `<RouteRow>` to `<tr>` (move that CSS under `StyledTable`) (2 instances).